frontend/pulse3d/src/components/interactiveAnalysis/WaveformGraph.js

```diff
@@ -953,29 +906,12 @@ export default function WaveformGraph({
   const movePeakValley = (keyCode) => {
     const { type, idx } = selectedMarkerToMove;
 
-    if (keyCode === 37) {
-      // 37 is left arrow key
-      if (type === "peak") {
-        const idxVal = peaks[idx];
-        peaks.splice(idx, 1, (idxVal -= 1));
-        setPeaks([...peaks]); // required to change dependencies
-      } else {
-        const idxVal = valleys[idx];
-        valleys.splice(idx, 1, (idxVal -= 1));
-        setValleys([...valleys]); // required to change dependencies
-      }
-    } else if (keyCode === 39) {
-      // 39 is right arrow key
-      if (type === "peak") {
-        const idxVal = peaks[idx];
-        peaks.splice(idx, 1, (idxVal += 1));
-        setPeaks([...peaks]); // required to change dependencies
-      } else {
-        const idxVal = valleys[idx];
-        valleys.splice(idx, 1, (idxVal += 1));
-        setValleys([...valleys]); // required to change dependencies
-      }
-    }
+    // Tanner (5/25/23): Currently assuming that only 37 and 39 will be passed in
```

Comment:
What happens if another key is typed?

Reply:
I'll have to check

Reply:
Should only call this function though if one of those two keys are pressed https://github.com/CuriBio/k8s_curi/pull/537/files/4b7c9ae835d7cecdeb83d2b1f2c9f72151e39786#diff-fc4024e23c0ac1f90740005d341ae9dbd8db7fdbfac7ce3c5cba660f2c97f59dR364